plugins/http_extract/src/http_extract/config.py

```
@@ -0,0 +1,168 @@
+        default=3, ge=0, le=10, description="Количество повторных попыток"
+    )
+    retry_delay: float = Field(
+        default=1.0, ge=0.1, le=60.0, description="Задержка между попытками"
+    )
+
+    # Обработка ошибок
+    ignore_http_errors: bool = Field(
+        default=False, description="Игнорировать HTTP ошибки"
+    )
+    valid_status_codes: list[int] = Field(
+        default_factory=lambda: [200], description="Валидные коды ответа"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)